setup/connector_prestashop/setup.py

```diff
@@ -2,5 +2,11 @@
 
 setuptools.setup(
     setup_requires=['setuptools-odoo'],
-    odoo_addon=True,
+    odoo_addon={
+        'external_dependencies_override': {
+            'python': {
+                'vcr': 'vcrpy==1.10.5',
```

Comment:
Thanks! We should avoid pinning versions here. Specifying a minimum required version is fine, while pinning must be never be done in OCA repos and done in customer projects only.

Reply:
This is done on purpose as upper versions don't work.

Reply:
I'm thinking that maybe we can just leave out the dependency as it's only for testing. What do you think?

Reply:
Ah, ok, if other versions don't work then fine. It must be considered as a bug in the module though, as depending on pinned, old, versions will create problems sooner or later.